### PR TITLE
Change documentation for googlehome

### PR DESCRIPTION
**Pull request by ludeeus**
**Description:**

Changes to the documentation for googlehome.

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#19971

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

**Review by klaasnicolaas — DISMISSED** [review]

File: source/_components/googlehome.markdown [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+layout: page
+title: "Google Home"
+description: "Instructions on how to connect to your Google Home device."
+date: 2018-11-04 00:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: google_home.png
+ha_release: 0.83
```
Comment (MartinHjelmare) [inline source/_components/googlehome.markdown]: 0.88

Comment (MartinHjelmare) [reply source/_components/googlehome.markdown]: The component is new but not the platform. 🤷‍♂️

File: source/_components/googlehome.markdown [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+    rssi_threshold:
+      description: The threshold for the RSSI signal of the device.
+      required: false
+      default: -70
+      type: integer
+    device_types:
+      description: Device types that will be tracked [see device types](#device_types), by default all types are tracked.
+      required: false
+      type: list
+{% endconfiguration %}
```
Comment (MartinHjelmare) [inline source/_components/googlehome.markdown:54]: We should add the possible items of the list here in `keys` I think.

File: source/_components/googlehome.markdown (hunk 1 above)
Comment (ludeeus) [reply source/_components/googlehome.markdown]: Was unsure about this, updated to 88

File: source/_components/googlehome.markdown [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+      description: The threshold for the RSSI signal of the device.
+      required: false
+      default: -70
+      type: integer
+    device_types:
+      description: Device types that will be tracked [see device types](#device_types), by default all types are tracked.
+      required: false
+      type: list
+      keys:
+        1:
```
Comment (MartinHjelmare) [inline source/_components/googlehome.markdown]: Maybe this just looks messy? If we want to keep it we have to quote the numbers. But I think we can revert to the previous way it was. Sorry.

Comment (ludeeus) [reply source/_components/googlehome.markdown]: This way is more correct according to the standard, but I agree that this looks messier 🤷‍♂️

Comment (ludeeus) [reply source/_components/googlehome.markdown]: Added quotes now, so we can see when it has rendered :)

Comment (MartinHjelmare) [reply source/_components/googlehome.markdown]: The previous way was better I think. I don't understand what 1, 2, 3 means.

**Review by MartinHjelmare — APPROVED** [review]

**Pull request merged**